cmd/metadata/metadata.go

```diff
@@ -52,6 +52,12 @@ func PackageMetadataCmd() *cobra.Command {
 				schemaFile = fmt.Sprintf("provider/cmd/pulumi-resource-%s/schema.json", providerName)
 			}
 
+			repoOwner := ""
+			githubSlugParts := strings.Split(repoSlug, "/")
+			if len(githubSlugParts) > 0 {
+				repoOwner = githubSlugParts[0]
+			}
```

Comment:
maybe overkill but should we account for the possibility of bad input, i.e. someone passing in the FQDN for their provider, which would result in a repoOwner of `https:` ?